### ISSUE TITLE
PS2 vs PS1 notation for power series

### ISSUE THREAD

**Issue by owensks** [TeamBasedInquiryLearning/calculus#100]
[Definition 9.1.1](https://teambasedinquirylearning.github.io/calculus/2023/PS1.html#def-PS1powerseries) defines a power series as centered at x=c, where the coefficients are a sequence {a_n}. I guess this makes sense because then the center is x=c, and the coefficients are a numeric sequence {a_n} like we've seen in all the previous sections. I guess "c" stands for _center_.

But then we get to [Remark 9.2.1](https://teambasedinquirylearning.github.io/calculus/2023/PS2.html#remark-51), which then switches what was the center x=c to be the center x=a, and switches the coefficient sequence {a_n} to be a coefficient sequence {c_n}. I guess "c" stands for _coefficient_.

Remark 9.2.1 uses coefficients c_n but then links back to Fact 8.7.1 which uses coefficients a_n. 

Long story short, I think it would probably be good to rewrite this section to agree with coefficients being {a_n} and power series defined by (x-c)^n. Having (x-a)^n with {c_n} sometimes but then (x-c)^n with {a_n} other times is not ideal.

**Comment by StevenClontz** [comment]
I have no opinion other than agreeing they should all be consistent in our book.

(And that I appreciate Kate's diligence in opening these issues and am very happy we have a collaborative team working on the library!)

**Comment by owensks** [comment]
So I spent some time thinking about this last night. To do IOC or ROC, students compute `\( \lim_{n \to \infty} \frac{a_{n+1}}{a_n}} \)` but in this case the a_n are really of the form `\( b_n = a_n (x-c)^n \) ` and our letters are confused, again. Perhaps it would make sense to call something a power function if it has the form \( b_n (x-c)^n \) and then to find the radius of convergence we compute `\( \lim_{n \to \infty} \frac{a_{n+1}}{a_n}} \)` where a_n is defined as `\( a_n = b_n (x-c)^n \)` 

Are students going to be confused about why the Ratio Test has a_n without any (x-c)^n, then power series are a_n (x-c)^n, but we're still doing the Ratio Test? Somehow we are composing a_n with a function that already contains a_n.

TL;DR
What do you think about defining power series as \series b_n (x-c)^n everywhere

**Comment by StevenClontz** [comment]
I don't think in practice when students see

$$\sum_{n=0}^\infty \frac{x^n}{n+1}$$

that they think "oh here we have $a_n=\frac{1}{n+1}$ as part of the power series definition". It only comes up once they're checking endpoints

$$\sum_{n=0}^\infty \frac{1}{n+1}$$

$$\sum_{n=0}^\infty \frac{(-1)^n}{n+1}$$

and in these cases the power series definition is irrelevant?

**Comment by owensks** [comment]
Sounds good. I'll make the changes to PS2 and push them. Pull them? Merge them? Click the buttons in the right spot them. 😁

**Comment by StevenClontz** [comment]
(As a mostly irrelevant aside: I would not be sad if we ditched endpoints of intervals of convergence, or pushed them to the back of the section so they can be skipped.)
